.changeset/ninety-nails-report.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
```

Comment:
@hashicorp/hds-engineering _technically_ this could be considered a breaking change, but we know that all of our consumers are on Ember versions greater than 3.24 so this shouldn't have a practical risk to it we release it as a patch (or minor). Any objections?

Reply:
I would argue that our consumers:
- or they're [already importing it in their package.json files](https://github.com/search?q=%22ember-named-blocks-polyfill%22++user%3Ahashicorp+path%3A*.json&type=Code&ref=advsearch&l=&l=)
- or they don't need it
So probably safe to release it as a patch. unless we want to add it to the batch of things we want to release as "major" (which is also an option I would consider)

In both cases, we should clearly document/publicize the change.

Reply:
that they can add this themselves if needed is a good point